src/calendar/day/period/index.js

```diff
@@ -83,24 +81,19 @@ class Day extends Component {
         return prev;
       }
 
-      const color = next.color;
-      if (next.status === 'NotAvailable') {
+      if (next.status === 'NotAvailable') { //???
```

Comment:
I think this is a leftover from `hotels-world-app` (WixApp Hotels module). This library started as a stripped out code from Hotels. We should remove it because it's WixApp specific, but we need to be sure we don't break Hotels. It's no longer maintained but it's still supported by our team (mainly Karolis).

Reply:
We can remove it in V2 so we won't break anyone